### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PentesterPriyanshu**
Some sentences earlier were less readable and had small grammatical errors. I have corrected all of that and made the readme file very readable for all users.

# Changes proposed ✍️

### What
- Updated the README file for improved readability and grammar.

### Why
- To ensure better comprehension and clarity for all users.

### How
- Made necessary changes to sentence structures and grammar errors in the README file.

## Checklist ✅
- [x] Label appropriately with `Documentation`.
- [ ] Add screenshots to the PR description for relevant changes.
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [x] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

**Comment by PentesterPriyanshu** [comment]
@joanagmaia  merge this request i have seen some errors so kindly merge it as i am also working on other issues also but some errors i have just encountered while reading some are making sentences confusing

**Comment by PentesterPriyanshu** [comment]
you can review it close it if not required i am already working on another issue

File: README.md [hunk 1]
```diff
@@ -40,18 +40,18 @@
...
+Crowd.dev is the Developer Data Platform(DDP) that allows companies to centralize all touch points developers have with their product and brand, whether in the community (e.g., Stack Overflow or Reddit), product (open-source or SaaS), or commercial channels (e.g., HubSpot).The platform pulls data from various sources, normalizes it, matches identities across platforms, and enriches it with third-party data. The result is a unified 360-degree view of the developers who engage with your product and community, the companies they work for, and their position in their personal customer journey.
```
Comment (joanagmaia) [inline README.md:43]: Could you keep Crowd.dev as crowd.dev?

File: README.md [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
+Crowd.dev is the Developer Data Platform(DDP) that allows companies to centralize all touch points developers have with their product and brand, whether in the community (e.g., Stack Overflow or Reddit), product (open-source or SaaS), or commercial channels (e.g., HubSpot).The platform pulls data from various sources, normalizes it, matches identities across platforms, and enriches it with third-party data. The result is a unified 360-degree view of the developers who engage with your product and community, the companies they work for, and their position in their personal customer journey. 
 
-crowd.dev is open-source, built with developers in mind, available for both hosted and self-hosted deployments, open to extensions, and offers full control over your data. 
+Crowd.dev is open-source, built with developers in mind, available for both hosted and self-hosted deployments, open to extensions, and offers full control over your data.
```
Comment (joanagmaia) [inline README.md:45]: Could you keep Crowd.dev as crowd.dev?

**Review by joanagmaia — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (PentesterPriyanshu) [reply README.md:45]: yes i am sorry yes i will keep that , i missed that crowd.dev is the origninal name of website

**Comment by PentesterPriyanshu** [comment]
@joanagmaia  if you want hover badges  i can also add that it will look like that
Screenshot 2023-10-26 at 17-03-02 Editing crowd dev_README md at patch-3 · PentesterPriyanshu_crowd dev

**Comment by joanagmaia** [comment]
@PentesterPriyanshu it all looks good to me. Could you fix the conflicts once again?
There have been a lot of contributions to README file

**Comment by PentesterPriyanshu** [comment]
@joanagmaia    now merge this otherwise we will face conflicts

**Review by joanagmaia — APPROVED** [review]

**Pull request merged**